### PR TITLE
refactor(lane_change): reducing clang-tidy warnings

### PR DESCRIPTION
**Pull request by zulfaqar-azmi-t4**
## Description

Simple refactoring to reduce clang-tidy warnings.

## Related links

**Parent Issue:**

- Link

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Review by maxime-clem — APPROVED** [review]

File: planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/scene.hpp [hunk 1]
```diff
@@ -127,7 +127,7 @@ class NormalLaneChange : public LaneChangeBase
   std::vector<double> calc_prepare_durations() const;
 
   lane_change::TargetObjects getTargetObjects(
```
Comment (mkquda) [inline planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/scene.hpp:129]: change to use snake_case?

File: planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/utils/utils.hpp [hunk 2]
```diff
@@ -85,7 +85,7 @@ std::vector<int64_t> replaceWithSortedIds(
 std::vector<std::vector<int64_t>> get_sorted_lane_ids(const CommonDataPtr & common_data_ptr);
 
 lanelet::ConstLanelets getTargetNeighborLanes(
```
Comment (mkquda) [inline planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/utils/utils.hpp:87]: change to use snake_case?

**Review by mkquda — APPROVED** [review]

File: planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/scene.hpp (hunk 1 above)
Comment (zulfaqar-azmi-t4) [reply planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/scene.hpp]: done https://github.com/autowarefoundation/autoware.universe/pull/9085/commits/46cf31901739f76a0b699edd9b542012d140611e

File: planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/utils/utils.hpp (hunk 2 above)
Comment (zulfaqar-azmi-t4) [reply planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/utils/utils.hpp]: done in https://github.com/autowarefoundation/autoware.universe/pull/9085/commits/46cf31901739f76a0b699edd9b542012d140611e

**Pull request merged**